Source/AppLovinAdapterConfiguration.swift

```diff
@@ -42,28 +42,15 @@ extension AppLovinAdapterConfiguration {
     
     /// The AppLovin SDK instance
     static var sdk: ALSdk? { AppLovinAdapter.sdk }
-    
-    static func sync() {
```

Comment:
I think this was still useful.
Can we keep this method but remove the `syncTestMode` one? I don't see `syncTestMode` doing nothing but logging some message to the console.

Reply:
you're right, I readded the sync() and remove syncTestMode() since test mode setting is being handled in the sdk init configuration now